### DISCUSSION TITLE
Unable to Access MIMIC-IV on BigQuery despite MIMIC-III Access and MIMIC-IV File Download

### DISCUSSION THREAD

**Discussion by ksmotty** [MIT-LCP/mimic-code#1824] [MIMIC-General] [answered]
Hello.

I am writing to inquire about an issue I am experiencing with accessing MIMIC-IV on BigQuery.

While I have successfully gained access to MIMIC-III（in 2020）and can download MIMIC-IV files(yesterday), I am unable to access MIMIC-IV on BigQuery.

I have confirmed that my PhysioNet account has been granted access to MIMIC-IV. However, when I try to open the data in BigQuery, I can only see "mimiciii" within "physionet-data" and there is no "mimiciv" folder. (Please see the attached screenshot for details.) 
I can download the MIMIC-IV files, but they are not appearing in BigQuery.

I have already completed the required CITI training, "Data or Specimens Only Research" course.
(Please see the attached screenshot for details.) 

I found a similar question in the Q&A forum (https://github.com/MIT-LCP/mimic-code/discussions/1453), and my situation seems to be similar to this user's.
I tried the solution suggested in the forum post, but unfortunately, it did not resolve the issue for me.

Could you please investigate this issue and advise on any further steps I need to take?

![image (3)](https://github.com/user-attachments/assets/e1279bfa-75b2-4d00-8d3f-e95560ee7c32)
![image (2)](https://github.com/user-attachments/assets/beea90d2-2a97-4294-81c3-10d290261f11)
![image (1)](https://github.com/user-attachments/assets/ac4567ce-4667-4048-88c0-f81b16a1a690)
![image](https://github.com/user-attachments/assets/d79beb70-c42d-4432-bb39-0bc235d65134)

**Comment by marymlucas** [comment, marked as answer]
Hi @ksmotty 
I think the issue may be that the current version of MIMIC-IV (v 3) is not yet available on BigQuery.  If you have access to v 2.2 (https://physionet.org/content/mimiciv/2.2/) then you should be able to see it on BigQuery. 

See issue https://github.com/MIT-LCP/mimic-code/issues/1803

**Reply by ksmotty** [reply]
Hi @marymlucas

Thank you for your reply.

I didn't know that MIMIC-IV (v2.3) is not yet available on BigQuery. I can now successfully access v2.2.

Thank you so much!

**Reply by tompollard** [reply]
This is correct, thanks @marymlucas! We will get the latest version of MIMIC onto BigQuery as soon as we can.
